internal/runners/packages/info.go

```diff
@@ -63,7 +64,12 @@ func (i *Info) Run(params InfoRunParams, nstype model.NamespaceType) error {
 		ns = ptr.To(model.NewNamespacePkgOrBundle(language, nstype))
 	}
 
-	packages, err := model.SearchIngredientsStrict(ns.String(), params.Package.Name, true, true, params.Timestamp.Time)
+	normalized, err := model.FetchNormalizedName(*ns, params.Package.Name)
+	if err != nil {
+		multilog.Error("Failed to normalize '%s': %v", params.Package.Name, err)
+	}
+
+	packages, err := model.SearchIngredientsStrict(ns.String(), normalized, false, false, params.Timestamp.Time) // ideally case-sensitive would be true (PB-4371)
```

Comment:
Since we're not returning an error when we attempt to fetch the normalized name we will need to check if the name is empty and pass the original. Otherwise, on error we are potentially just passing an empty string. If that's not how `FetchNormalizedName` works can you add a comment?

Reply:
You are correct that the default on error should be the user-provided name. Thanks for catching this.